perceval/backends/core/git.py

```diff
@@ -348,6 +355,9 @@ def setup_cmd_parser():
         group.add_argument('--latest-items', dest='latest_items',
                            action='store_true',
                            help="Fetch latest commits added to the repository")
+        group.add_argument('--no-update', dest='no_update',
```

Comment:
Probably, `--latest-items` and `--no-update` should be mutually exclusive. To use both at the same time doesn't make sense.